### ISSUE TITLE
[demonoid] String was not recognized as a valid DateTime. (Config)

### ISSUE THREAD

**Issue by igreulich** [Jackett/Jackett#1725]
Jackett version:  
-------------------------------
```
Jackett Version 0.8.105.0
```

Mono version (if not using Windows):  
-------------------------------
```
Mono JIT compiler version 5.0.1.1 (2017-02/5077205 Wed May 31 14:47:04 BST 2017)
Copyright (C) 2002-2014 Novell, Inc, Xamarin Inc and Contributors. www.mono-project.com
	TLS:           normal
	SIGSEGV:       altstack
	Notification:  kqueue
	Architecture:  amd64
	Disabled:      none
	Misc:          softdebug 
	LLVM:          supported, not enabled.
	GC:            sgen (concurrent by default)
```
When testing Demonoid, I get the following error:
```
An error occured while updating this indexer
String was not recognized as a valid DateTime.
```
Here is the relevant log info: (The rest of the log was the Demonoid html/css)
```
2017-08-26 13:14:33.7131 Error Error on 20170826131433 for Demonoid.txtSystem.FormatException: String was not recognized as a valid DateTime.
  at System.DateTimeParse.ParseExact (System.String s, System.String format, System.Globalization.DateTimeFormatInfo dtfi, System.Globalization.DateTimeStyles style) [0x0002a] in <ab2f5ad0f8c84dfe97dfd6a06a64cf44>:0 
  at System.DateTime.ParseExact (System.String s, System.String format, System.IFormatProvider provider) [0x00008] in <ab2f5ad0f8c84dfe97dfd6a06a64cf44>:0 
  at Jackett.Indexers.Demonoid+<PerformQuery>c__async1.MoveNext () [0x001f8] in <ac2251915c0442d0af0ee8832f707ded>:0 
```

**Comment by metaMMA** [comment]
Same issue.

Jackett Version: 0.8.105.0 
Mono Version: 4.6.2

**Comment by kaso17** [comment]
fixed via 441aad5a18f2171a6eca69596c08340e1b2410ca

**Comment by kaso17** [comment]
fixed in the latest pre-release
